src/styles/index.scss

```diff
@@ -1,3 +1,105 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.cards {
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  &__foto {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-size: 12px;
```

Comment:
The font-size property is redundant here since it's already defined for the `.cards` block on line 19. To follow DRY (Don't Repeat Yourself) principles, avoid declaring the same property with the same value if it can be inherited from a parent element.